decidim-core/app/cells/decidim/upload_modal/modal.erb

```diff
@@ -1,25 +1,24 @@
-<div class="reveal upload-modal" id="<%= modal_id %>" data-reveal>
-  <div class="reveal__header">
-    <h3 class="reveal__title" data-addlabel="<%= button_label %>" data-editlabel="<%= button_edit_label %>"><%= label %></h3>
-    <button class="close-button" data-close aria-label="<%= t("layouts.decidim.upload_modal.close_modal") %>"
-      type="button">
-      <span aria-hidden="true">&times;</span>
-    </button>
-  </div>
+<%= decidim_modal id: modal_id, class: "upload-modal" do %>
+<div data-dialog-container>
+  <%= icon "upload-cloud-2-line" %>
+  <div>
+    <h2 id="dialog-title-<%= modal_id %>" data-dialog-title data-addlabel="<%= button_label %>" data-editlabel="<%= button_edit_label %>"><%= label %></h2>
 
-  <div class="help">
-    <span><%= explanation %></span>
-    <ul>
-      <% help_messages.each do |message| %>
-        <li><%= message %></li>
-      <% end %>
-    </ul>
-  </div>
-  <div class="row dropzone-container" data-name="<%= attribute %>">
-    <%= tag.div class: "upload-items", data: {
+    <div data-name="<%= attribute %>" class="upload-modal__dropzone-container" data-dropzone>
```

Comment:
After I had validation errors in the modal, I could click "Next" at the bottom of the modal.

After that I could submit the verification form and I will see this message:
![Success after errors](https://user-images.githubusercontent.com/864340/199264478-62101c89-c3ce-4950-9652-ab6259bb08a1.png)

Maybe we shouldn't allow submitting the form in case we have errors with the files?

This is also how the current upload modals work. When we have errors in the upload modal, it will prevent submitting the form above it and it will show an error text next to the button that opens the modal.

Reply:
After the above error and the success message, the image was actually uploaded when I go back to the verification view:
![Image uploaded](https://user-images.githubusercontent.com/864340/199265285-793b57f3-2945-47f1-a3af-061d41133a62.png)

In case the image does not pass the upload validations, we should not allow saving it.

Reply:
I cannot click the text "Drop files here or click to upload":
![Click to upload](https://user-images.githubusercontent.com/864340/199265667-1eddf9d1-d216-4a05-b22a-9312827172d1.png)

I would expect that if the text says "click to upload", I would be able to click anywhere in the grey area to open the file selector.

Reply:
When I bring a file on top of the dropzone and then take the file back to its original location, the dropzone stays active even after I tried to "cancel" the drop:
![Dropzone stays active](https://user-images.githubusercontent.com/864340/199265981-6f65106c-d61a-4a8b-97a3-880a7d6ec4c2.png)

Reply:
Is it intentional that I can upload multiple files when I believe the verification should only allow uploading a single file?
![Uploading multiple files is possible](https://user-images.githubusercontent.com/864340/199269339-8636b43a-27a9-437a-8a77-4f2db3462be3.png)
![Multiple files selected](https://user-images.githubusercontent.com/864340/199269434-af08ae54-7ec7-4c66-afc5-466222c5aff5.png)

After I submit the form and come back, it only picks the last file I uploaded:
![Last file only](https://user-images.githubusercontent.com/864340/199269601-090b74fd-84f0-418c-b3ad-a397423d9780.png)